src/app/helpers/controller/bus-mixin.js

```diff
@@ -55,9 +55,32 @@ export class Bus {
 */
 export default (superclass) => class extends superclass {
 
+    constructor(...args) {
```

Comment:
This is probably the only file that I had hard time reading (possibly because it's a bus)

Reply:
I mis-documented this as I was writing it, which is likely one reason it was hard to read. I have given it correct documentation.